public/json/products.json

```diff
@@ -34,6 +34,21 @@
                     "price" : "120"
                 }
             ]
+        },
+        {
+            "type" : "Ring Album",
```

Comment:
Instead of Ring Album make it simply "albums", and with in all the different types of albums